src/Tabs/Tabs.js

```diff
@@ -377,4 +377,3 @@ class Tabs extends Component {
 }
 
 export default withWidth()(Tabs);
-export { Tabs as PureTabs }; // for testing purposes
```

Comment:
@oliviertassinari Got it - thanks!  I was trying to `dive()` against the rendered `<Tabs>` but failing because it returned a function.  I see why your way is working and mine wasn't.  I appreciate the insight! :clap: